src/test/java/org/jabref/logic/integrity/AuthorsContainedCheckerTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.logic.integrity;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class AuthorsContainedCheckerTest {
+
+    private final AuthorsContainedChecker checker = new AuthorsContainedChecker();
+
+    @Test
+    void booktitleDoesNotAcceptIfItContainsAuthor() {
+        assertNotEquals(Optional.empty(), checker.checkValue("Service-Oriented Computing - {ICSOC}, Fifth International Conference, Vienna, Austria, Proceedings"));
```

Comment:
`ICSOC` is NOT an author - it is an abbreviation of the conference.

Maybe, the whole author check should be removed!